site/app/(platform)/agencies/_components/property-form.tsx

```
@@ -0,0 +1,66 @@
+"use client";
+import { stepSchemas } from "@/types/property";
+import { MultiStepForm } from "./step-form";
+import { House, Map, User, Wallet, Camera, File } from "lucide-react";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)